### PR TITLE
Update restore-statements-headeronly-transact-sql.md

### PR DESCRIPTION
**Pull request by 0x7FFFFFFFFFFFFFFF**
Add new columns. See more discussions here: https://github.com/MicrosoftDocs/sql-docs/issues/7879

### CODE REVIEW

**Comment by Court72** [comment]
@MikeRayMSFT 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/statements/restore-statements-headeronly-transact-sql.md [hunk 1]
```diff
@@ -141,6 +141,8 @@ FROM <backup_device>
...
 |**EncryptorThumbprint**|**varbinary(20)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The thumbprint of the encryptor which can be used to find certificate or the asymmetric key in the database. When the backup was not encrypted, this value is NULL.|  
 |**EncryptorType**|**nvarchar(32)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The type of encryptor used: Certificate or Asymmetric Key. When the backup was not encrypted, this value is NULL.|
+|**LastValidRestoreTime**|**datetime**|**Applies beginning with**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)]. The last valid restore time.|
```
Comment (rwestMSFT) [inline docs/t-sql/statements/restore-statements-headeronly-transact-sql.md]:
```suggestion
|**LastValidRestoreTime**|**datetime**|**Applies to**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)] and later versions. The last valid restore time.|

```

File: docs/t-sql/statements/restore-statements-headeronly-transact-sql.md [hunk 2]
```diff
@@ -141,6 +141,8 @@ FROM <backup_device>
...
 |**EncryptorThumbprint**|**varbinary(20)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The thumbprint of the encryptor which can be used to find certificate or the asymmetric key in the database. When the backup was not encrypted, this value is NULL.|  
 |**EncryptorType**|**nvarchar(32)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The type of encryptor used: Certificate or Asymmetric Key. When the backup was not encrypted, this value is NULL.|
+|**LastValidRestoreTime**|**datetime**|**Applies beginning with**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)]. The last valid restore time.|
+|**TimeZone**|**nvarchar(32)**|**Applies beginning with**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)]. The time zone of the server from which the backup was taken.|
```
Comment (rwestMSFT) [inline docs/t-sql/statements/restore-statements-headeronly-transact-sql.md]:
```suggestion
|**TimeZone**|**nvarchar(32)**|**Applies to**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)] and later versions. The time zone of the server from which the backup was taken.|
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]

File: docs/t-sql/statements/restore-statements-headeronly-transact-sql.md [hunk 3]
```diff
@@ -141,6 +141,9 @@ FROM <backup_device>
...
 |**EncryptorThumbprint**|**varbinary(20)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The thumbprint of the encryptor which can be used to find certificate or the asymmetric key in the database. When the backup was not encrypted, this value is NULL.|  
 |**EncryptorType**|**nvarchar(32)**|**Applies to**: [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] ([!INCLUDE[ssSQL14](../../includes/sssql14-md.md)] (CU1) through current version.<br /><br /> The type of encryptor used: Certificate or Asymmetric Key. When the backup was not encrypted, this value is NULL.|
+|**LastValidRestoreTime**|**datetime**|**Applies to**: [!INCLUDE [sssql22-md](../../includes/sssql22-md.md)] and later versions. The last valid restore time.|
+
```
Comment (rwestMSFT) [inline docs/t-sql/statements/restore-statements-headeronly-transact-sql.md]:
```suggestion
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]

**Comment by rwestMSFT** [comment]
Replaced by internal PR.

**Pull request closed without merging**